packages/maskbook/src/plugins/VCent/base.ts

```diff
@@ -0,0 +1,15 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { PLUGIN_ID, PLUGIN_ICON, PLUGIN_NAME, PLUGIN_DESCRIPTION } from './constants'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    icon: PLUGIN_ICON,
+    name: { fallback: PLUGIN_NAME },
+    description: { fallback: PLUGIN_DESCRIPTION },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
```

Comment:
IIRC this plugin is contributed by the community right?

Reply:
@yisiliu Can you provide the provider's information?

Reply:
it was https://github.com/DimensionDev/Maskbook/pull/2683

Reply:
@UncleBill Take a look?

Reply:
@guanbinrui I will update later.

Reply:
Updated the publisher as @rob-lw